### DISCUSSION TITLE
Android app found : Validating Authenticity

### DISCUSSION THREAD

**Discussion by anand3132** [AzuraCast/AzuraCast#7634] [General]
do you have any android app i noticed two in play store 
1: https://play.google.com/store/apps/details?id=gr.techzombie.azuracastadmin&hl=en_IN
2: https://play.google.com/store/apps/details?id=gr.techzombie.radiounicorn&hl=en_IN

**Comment by BusterNeece** [comment]
We don't have any official app released for Android or iOS. The web panel on every AzuraCast installation is meant to be responsive, so you could access it via a mobile device if needed (though your experience is likely better on desktop/tablet devices).

I believe the first app you listed is just using our API to do its administrative work, and the second isn't an administrator app at all, but rather a player app for a specific radio station.

**Reply by anand3132** [reply]
Thanks for the replay ..!! its a nice app you brought in ..!!
Small Suggestion : On creating new station > after adding music > if I forget to create a playlist > station will not start   you can add one error toss message telling to create playlist before starting the station..!!

**Comment by DJMic** [comment]
See here : https://github.com/AzuraCast/AzuraCast/discussions/7198
